app/models/product_import/entry_processor.rb

```diff
@@ -149,6 +149,7 @@ def save_existing_inventory_item(entry)
       end
     end
 
+    # rubocop:disable Metrics/AbcSize
```

Comment:
Curious, I don't see how this change increased the [ABC size](https://www.rubydoc.info/gems/rubocop/0.27.0/RuboCop/Cop/Metrics/AbcSize), so I'm not sure if this `disable` comment is necessary.

If it is, it would be better to add it to the `.rubocop_todo.yml` file, to remind us to improve it next time we work on the file.

Reply:
It does change the Abc size somehow, I just tested. I can't see a quick and easy solution to fix that, so better to add it to `.rubocop_todo.yml`. You can run the following command to do so : 
```
bundle exec rubocop --regenerate-todo --no-auto-gen-timestamp
```

Reply:
@wandji20 you need to remove the rubocop comments before updating the todo files, otherwise it gets ignored.